src/debug.h

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright libCellML Contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#pragma once
+
+#include <iostream>
+#include <sstream>
+
+namespace libcellml {
+
+struct dbg
```

Comment:
No it can't be consolidated with the `debug()` in test utilities.  I find it kind of useful but still has the same problem of leaving `#include <iostream>` around.

Reply:
At least, it would be nice to have `dbg` renamed to `Debug`...